src/org/traccar/geofence/GeofenceCircle.java

```diff
@@ -0,0 +1,82 @@
+package org.traccar.geofence;
+
+import java.text.DecimalFormat;
+import java.text.ParseException;
+
+import org.traccar.helper.DistanceCalculator;
+
+public class GeofenceCircle extends GeofenceGeometry {
+
+    private double centerLatitude;
+    private double centerLongitude;
+    private double radius;
+
+    public GeofenceCircle() {
+        super();
```

Comment:
There is no need to call `super` explicitly. Same applies to other constructors.